ga/developer/kernel/Dockerfile

```diff
@@ -19,9 +19,12 @@ LABEL maintainer="Arthur De Magalhaes <arthurdm@ca.ibm.com> (@arthurdm)"
 RUN apt-get update \
     && apt-get install -y --no-install-recommends unzip \
     && rm -rf /var/lib/apt/lists/*
```

Comment:
this line needs a `\` character at the end.  that is failing the PR build.

Reply:
Fixed in the new push.  Good catch.